.ci/config/matrices.yaml

```diff
@@ -29,8 +29,9 @@ engine-rest:
   - 'engine-rest-unit-cxf'
   - 'engine-rest-unit-compatibility-wildfly'
   - 'engine-rest-IT-embedded-wildfly'
```

Comment:
💭 Should we add these stages for Jakarta as well?

Reply:
The second one looks like something for the QA ticket.
We could add a `compatibility-wildfly` in Jakarta but it would essentially test the same Resteasy version.
We use Resteasy 6.2.3, Wildfly 27 uses 6.2.1.

Reply:
* `engine-rest-IT-embedded-wildfly`: the question is: do we want to support this use case with Wildfly 27? If not, we need to make this transparent in the docs. Can we take this question to the sync meeting?
* I think setting the foundation for `compatibility-wildfly` makes sense even if it's the same right now. Since we automatically add the latest Wildfly version with future Camunda releases, this becomes important soon, and we might miss it if we don't act now. Maybe we can set the foundation in the pom now, add a commented stage to the `matrices.yaml` that explains when to comment it in, and references a to-be-created follow-up ticket with the label for the future Camunda 7 minor version in which we potentially add Wildfly 28.